packages/lit-html/src/test/lit-html_test.ts

```diff
@@ -1875,6 +1875,18 @@ suite('lit-html', () => {
       assert.isOk(event);
     });
 
+    test('event listeners can see events fired in element directives', () => {
+      let event = undefined;
+      const listener = (e: Event) => {
+        event = e;
+      };
+      render(
+        html`<div @test-event=${listener} ${fireEvent()}></div>`,
```

Comment:
If the parts were ordered oppositely would this still work?

Reply:
No, as you know expressions are evaluated in order, so if the directive synchronously fires an event it would not be heard by a later expression that added an event listener, and we're not going to change any of that.

Frankly I'm not sure of the point of this test; it was added in @justinfagnani's original PR that added directives. This is just closing out a TODO.